src/app/pages/OnDemandRadioPage/index.jsx

```diff
@@ -145,6 +147,8 @@ const OnDemandRadioPage = ({ pageData }) => {
     translations,
   );
 
+  const radioScheduleData = path(['radioScheduleData'], pageData);
```

Comment:
I think line 234 will throw a `TypeError` if `radioSchedule` is undefined

```suggestion
  const radioScheduleData = path(['radioScheduleData'], pageData) || [];
```

(or `pathOr`)

Or if we're assuming it's guaranteed to be an array (looks like it is from the `getInitialData` code at least) we should add it as a required propType